src/webargs/flaskparser.py

```diff
@@ -117,6 +116,18 @@ def get_default_request(self):
         return flask.request
 
 
+class FlaskAsyncParser(FlaskParser, asyncparser.AsyncParser):
+    """Flask request argument parser for async views (supported on Flask
+    2.0 and higher).
+    """
+
+    pass
```

Comment:
> The only other thing that I think we need is some kind of update to the documentation to note that using `webargs` on a flask async view requires `use_args_async`, `use_kwargs_async`, `async_parser`, or `FlaskAsyncParser`. Could you add some lines to docs/framework_support.rst` to cover this? It doesn't need to be comprehensive -- just something similar to the flask example which is already there, but for an async route.

Yeah, I can do that.

Reply:
> This is a cosmetic nitpick, but I'd prefer we leave the body empty rather than an unnecessary `pass`.

The `pass` statement is needed for empty classes, otherwise an error will occur.

Reply:
On what python versions? I find that this works fine on every interpreter I test:

```
$ cat foo.py
class A:
    """hi"""
$ for py in python3.6 python3.7 python3.8 python3.9 python3.10 python3.11 pypy3; do $py foo.py; done
# no output, no errors
```

The class has a body if it has a docstring, which is sufficient.

Reply:
Oh, right. Thanks!